tests/cypress/support/commands_projects.js

```diff
@@ -37,6 +37,33 @@ Cypress.Commands.add(
     },
 );
 
+Cypress.Commands.add('deletingCreatedProjects', (response, projectsToDelete) => {
```

Comment:
The same comments like for users/tasks/organizations

Reply:
Done